go.sum

```diff
@@ -1,4 +1,6 @@
 cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
+github.com/99designs/go-keychain v0.0.0-20191008050251-8e49817e8af4 h1:/vQbFIOMbk2FiG/kXiLl8BRyzTWDw7gX/Hz7Dd5eDMs=
```

Comment:
What does this change have to do with this PR?

Reply:
Believe this is something @jackzampolin wanted to remove that annoying warning notice whenever you build sdk on Mac